extensions/antennas.sigmf-ext.md

```diff
@@ -0,0 +1,36 @@
+# Antenna Extension
+
+The `antenna` namespace extension defines one object for static antenna parameters extending `global` and one object for dynamic antenna parameters extending `annotations`.
+
+## Global
+
+### Antenna Object
+The `Antenna` object contains the following name/value pairs:
+
+|name|required|type|unit|description|
+|----|--------------|-------|-------|-----------|
+|`model`|true|string|N/A|Antenna make and model number. E.g. `"ARA CSB-16"`, `"L-com HG3512UP-NF"`.|
+|`type`|false|string|N/A|Antenna type. E.g. `"dipole"`, `"biconical"`, `"monopole"`, `"conical monopole"`.|
+|`low_frequency`|false|float|Hz|Low frequency of operational range.|
+|`high_frequency`|false|float|Hz|High frequency of operational range.|
+|`gain`|false|float|dBi|Antenna gain in direction of maximum radiation or reception.|
+|`horizontal_gain_pattern`|false|array of floats|dBi|Antenna gain pattern in horizontal plane from 0 to 359 degrees in 1 degree steps.|
+|`vertical_gain_pattern`|false|array of floats|dBi|Antenna gain pattern in vertical plane from -90 to +90 degrees in 1 degree steps.|
+|`horizontal_beam_width`|false|float|degrees|Horizontal 3-dB beamwidth.|
+|`vertical_beam_width`|false|float|degrees|Vertical 3-dB beamwidth.|
+|`cross_polar_discrimination`|false|float|N/A|Cross-polarization discrimination.|
+|`voltage_standing_wave_ratio`|false|float|volts|Voltage standing wave ratio.|
+|`cable_loss`|false|float|dB|Cable loss for cable connecting antenna and preselector.|
+|`steerable`|false|boolean|N/A|Defines if the antenna is steerable or not.|
+|`mobile`|false|boolean|N/A|Defines if the antenna is mobile or not.|
+
+## Annotations
+
+### DynamicAntennaSettings Object
```

Comment:
I'm interested in your reasoning for creating a new namespace, `DynamicAntennaSettings`, for the fields that go in annotations. I think it would be simpler for them all to be `antenna`, kind of like how we have `core` pairs for all objects.

Reply:
You bring up a great point, we pulled this out of our own namespace extension but didn't properly format it as its own standalone namespace. We'll do a bit of refactoring and update this PR.